plugins/arDominionPlugin/css/less/jobs.less

```diff
@@ -48,7 +48,7 @@
   margin: 5px 5px 0px 5px;
   padding: 5px;
   background-color: #f2f2f2;
-  border-radius: 10px;
+  border-radius: 4px;
   color: @black;
   font-family: Courier New;
   max-height: 400px;
```

Comment:
I'd prefer no to use `max-height`, there's nothing wrong with having the user scrolling down the page, unless you expect that the contents inside the log are going to be huge?

Also, do you think we should use `white-space: nowrap;` together with `overflow: auto;` so the text is not wrapped but it can be horizontally scrolled when needed?